### PR TITLE
update some entries in glossary

### PR DESCRIPTION
**Pull request by tdykstra**
Scott Hanselman had some suggestions and I've tried to implement them -- @mairaw @BillWagner @rpetrusha what do you think of these changes?

### CODE REVIEW

File: docs/standard/glossary.md [hunk 1]
```diff
@@ -193,9 +193,9 @@ You can think of this technology as a persistent JIT compiler. It usually compil
 
 ## package
 
-For a package-based target framework, a NuGet package that contains an assembly of the same name.
+A NuGet package &mdash; or just a package &mdash; is a *.zip* file with one or more asemblies of the same name along with additional metadata such as the author name.
```
Comment (mairaw) [inline docs/standard/glossary.md]: typo: asemblies

File: docs/standard/glossary.md [hunk 2]
```diff
@@ -193,9 +193,9 @@ You can think of this technology as a persistent JIT compiler. It usually compil
...
 
-The package is a *.zip* file wih  a *.nupkg* extension that may contain assets (such as *.dll* files and *.xml* files) for use with multiple frameworks and versions. When installed in an app or library, the appropriate assets are selected based on the target framework specified by the app or library. The assets that define the interface are in the *ref* folder, and the assets that define the implementation are in the *lib* folder.
+The *.zip* file has a *.nupkg* extension and may contain assets (such as *.dll* files and *.xml* files) for use with multiple frameworks and versions. When installed in an app or library, the appropriate assets are selected based on the target framework specified by the app or library. The assets that define the interface are in the *ref* folder, and the assets that define the implementation are in the *lib* folder.
```
Comment (mairaw) [inline docs/standard/glossary.md]: should we replace parentheses with commas? Acrolinx usually complains about those.

**Review by mairaw — COMMENTED** [review]
LGTM, modulo some comments/questions. Also, please remove the BOM from the beginning of the file, otherwise the metadata section doesn't render correctly.

File: docs/standard/glossary.md [hunk 3]
```diff
@@ -35,6 +35,12 @@ A cross-platform, high-performance, open source implementation of ASP.NET built
...
 
+## assembly
+
+A *.dll* file that contains a collection of APIs that can be called by apps or other assemblies.
+
+A .NET assembly is a collection of types. An assembly includes interfaces, classes, structures, enumerations, and delegates. See also [library](#library).
+
 ## CLR
 
 Common language runtime.
```
Comment (scottaddie) [inline docs/standard/glossary.md]: Common language runtime --> Common Language Runtime

**Review by scottaddie — APPROVED** [review]
A few suggestions to consider. Also, it may eliminate confusion if we use ".NET Core" instead of the ".NET Core framework" heading. It's too close to ".NET Framework".

File: docs/standard/glossary.md [hunk 4]
```diff
@@ -35,6 +35,12 @@ A cross-platform, high-performance, open source implementation of ASP.NET built
 
 See [ASP.NET Core](/aspnet/#pivot=core).
 
+## assembly
+
+A *.dll* file that contains a collection of APIs that can be called by apps or other assemblies.
```
Comment (tdykstra) [inline docs/standard/glossary.md]: What is the context in which it's common to use "binary" in the sense of "assembly"?

Comment (scottaddie) [inline docs/standard/glossary.md]: I often hear folks referring to the *.dll* files in the bin folder as their project's binaries after they've built a project in VS.

Comment (mairaw) [inline docs/standard/glossary.md]: I see that you already added. We even have that in the CLI commands reference.

https://github.com/dotnet/docs/search?l=Markdown&q=binaries&type=&utf8=%E2%9C%93

**Pull request merged**